@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -1023,7 +1065,7 @@ export class VisibleTileSet {
             );
 
             const nextLevelDiff = Math.abs(childTileKey.level - dataZoomLevel);
-            if (childTile !== undefined && childTile.hasGeometry) {
+            if (childTile !== undefined && childTile.hasGeometry && !childTile.delayRendering) {
                 // childTile has geometry, so can be reused as fallback
```

Comment:
Please adjust the comment:
`childTile has geometry and was/can be uploaded to the GPU, so we can use it as fallback`